### PR TITLE
[wtforms] Strip leading/trailing whitespace

### PR DESCRIPTION
**Pull request by john-bodley**
Somewhat related to https://github.com/apache/incubator-superset/pull/5445 WTForms does not strip leading and trailing whitespace from fields which means that the database records may contain whitespace only strings (which look empty) as opposed to NULLs. 

Per [this](https://stackoverflow.com/questions/26232165/automatically-strip-all-values-in-wtforms) suggestion, this PR monkey patches the `bind_field` to ensure that any leading or trailing whitespace is stripped.

**Before**
before

**After**
after

to: @graceguo-supercat @michellethomas @mistercrunch @xtinec

### CODE REVIEW

File: superset/migrations/versions/d94d33dbe938_form_strip.py [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+    for table in tables:
+        for record in session.query(table).all():
+            for col in record.__table__.columns.values():
+                if not col.primary_key:
+                    value = getattr(record, col.name)
+
+                    if isinstance(value, str) and value.strip() == '':
+                        setattr(record, col.name, None)
+
+        session.commit()
```
Comment (graceguo-supercat) [inline superset/migrations/versions/d94d33dbe938_form_strip.py]: you want to commit change for every table? why not 1 commit for all tables?

Comment (john-bodley) [reply superset/migrations/versions/d94d33dbe938_form_strip.py]: @graceguo-supercat this follows @mistercrunch's recommendation from https://github.com/apache/incubator-superset/pull/5445.

Comment (graceguo-supercat) [reply superset/migrations/versions/d94d33dbe938_form_strip.py]: thanks for explanation!

**Review by graceguo-supercat — APPROVED** [review]

File: superset/migrations/versions/d94d33dbe938_form_strip.py (hunk 1 above)
Comment (mistercrunch) [reply superset/migrations/versions/d94d33dbe938_form_strip.py]: could be good to log status here periodically "Processing table: {table}" and things like:
```python
if i % 1000 = 0:
    session.commit()
    print("Processed {i} rows")
```

**Pull request merged**